config/config.default.json

```diff
@@ -14,7 +14,9 @@
     },
     "styles": {
       "stage": 3,
-      "importFrom": [],
+      "importFrom": [
+        "_custom-media.css"
```

Comment:
Unlike `custom-properties`, which can be declared once in `:root` of a single CSS file, `custom-media` must be declared in each CSS file. Instead of having to do `@import` for `custom-media` in each CSS file it makes sense to use `importFrom` to import `custom-media` into all CSS files at build time.